dicomexporter/exporter.py

```diff
@@ -25,12 +27,28 @@ def convertDICOMVolumeToVTKFile(
     """
     Converts DICOM files in a directory into a VTK file (.vti or .vtkjs)
     """
-    # Test output_file_path #
-    if not overwrite and os.path.exists(output_file_path):
-        print('Output file already exist', output_file_path, 
-            '\nIf you want to overwrite the file add the \'--overwrite\' flag')
+    _, file_extension = os.path.splitext(output_file_path)
+
+    if not file_extension in ALLOWED_EXTENSIONS:
+        print('Unknown file extension', file_extension)
         return False, None
 
+    # Test output_file_path #
+    if os.path.exists(output_file_path):
+        if not overwrite:

```

Comment:
@laurennlam we came up with the "NIP" (i.e. not important) prefix, to these kind of "preference"/"not important" remarks. you avoid therefore the long "disclaimer speech" :-)

And for what it worthes (i.e. NIP), I personally like to handle errors first rather than later :-)

Reply:
Thank you for the remark. Actually the else branch isn't even necessary, so I removed it. Makes it more readable that way.